File: articles/knowledge-architecture-in-governance-systems/cpp/governance_architecture_metrics.cpp
#include <iostream>
#include <iomanip>

int main() {
    double objects = 14.0;
    double relationships = 15.0;

    std::cout << std::fixed << std::setprecision(3);
    std::cout << "Metadata coverage: " << 13.0 / objects << "\n";
    std::cout << "Accountability context coverage: " << 11.0 / objects << "\n";
    std::cout << "Equity context coverage: " << 9.0 / objects << "\n";
    std::cout << "Relationship traceability: " << 14.0 / relationships << "\n";
    std::cout << "Underspecified relationship risk: " << 1.0 / relationships << "\n";

    return 0;
}

File: articles/knowledge-architecture-in-governance-systems/fortran/governance_architecture_metrics.f90
program governance_architecture_metrics
  implicit none

  real :: objects
  real :: relationships

  objects = 14.0
  relationships = 15.0

  print *, "Governance Knowledge Architecture Metrics"
  print *, "Metadata coverage: ", 13.0 / objects
  print *, "Accountability context coverage: ", 11.0 / objects
  print *, "Equity context coverage: ", 9.0 / objects
  print *, "Relationship traceability: ", 14.0 / relationships
  print *, "Underspecified relationship risk: ", 1.0 / relationships
end program governance_architecture_metrics
